spring-cloud-gcp-starters/spring-cloud-gcp-starter-data-datastore/pom.xml

```diff
@@ -23,5 +23,9 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-gcp-data-datastore</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-actuator</artifactId>
```

Comment:
It needs to remain optional and up to the user to decide whether to include.

Reply:
Why add it here at all? The sample doesn't use the starter doesn't use the actuator. I think this is masking an issue in the autoconfiguration.